Makefile.work

```diff
@@ -340,6 +352,11 @@ showtag:
 	@echo $(SLAVE_IMAGE):$(SLAVE_TAG)
 	@echo $(SLAVE_BASE_IMAGE):$(SLAVE_BASE_TAG)
 
+initpde:
+	$(QUIET)for submodule in $(PDESUBMODULES) ; do \
```

Comment:
Where is `QUIET` defined?

Reply:
I will fix the typo.

Reply:
Can you check the fix on the latest commit ?

Geans